src/app/shared/services/dynamic-data/dynamic-data.service.ts

```diff
@@ -147,6 +147,18 @@ export class DynamicDataService extends AsyncServiceBase {
     return { collectionName };
   }
 
+  /** Remove user writes on a flow to return it to its original state */
+  public async resetOverwrites(flow_type: FlowTypes.FlowType, flow_name: string) {
```

Comment:
nit(non-blocking)
nice method to have, although I can imagine some confusion between clearCache and resetOverwrites methods (in fact the remote-asset services uses its own `clearCache` method to call this reset method). I think probably best to combine into a single `reset` method. Currently I can't think of any use case where we would want to delete and not return to original state, but could be an optional parameter/option if required.

nit(non-blocking)
Should move public method above private